src/go_to_class/wdGoToClass/$construct.omh

```diff
@@ -10,6 +10,14 @@ Do ilLibraries.$sendall($ref.Name.$assign(low(ilLibraries.Name)))
 Do ilLibraries.$search($ref.Name=low($clib().$name),1,0,0,0) Returns ilLibraries.$line
 Do ilLibraries.$remove(0)
 
-Calculate ilLibraries.$line as pick(isclear(ciCurrentLibraryLine),ciCurrentLibraryLine,1)
+If isclear(ciCurrentLibraryLine)
```

Comment:
Let's eliminate `ciCurrentLibraryLine` altogether and just use `$class.$userinfo`.